### ISSUE TITLE
cuda12 support

### ISSUE THREAD

**Issue by tlambert03** [scopetools/cudadecon#28]
Trying to update the feedstock for cuda12 revealed that there are some things in geometryTransform.cu that are no longer compatible with cuda12:

```
/home/conda/feedstock_root/build_artifacts/cudadecon_1685986977889/work/src/geometryTransform.cu(180): error: texture is not a template

/home/conda/feedstock_root/build_artifacts/cudadecon_1685986977889/work/src/geometryTransform.cu(213): error: no instance of overloaded function "tex3D" matches the argument list
            argument types are: (<error-type>, float, float, float)

/home/conda/feedstock_root/build_artifacts/cudadecon_1685986977889/work/src/geometryTransform.cu(253): error: no instance of overloaded function "tex3D" matches the argument list
            argument types are: (<error-type>, float, float, float)

/home/conda/feedstock_root/build_artifacts/cudadecon_1685986977889/work/src/geometryTransform.cu(274): error: identifier "cudaBindTextureToArray" is undefined

/home/conda/feedstock_root/build_artifacts/cudadecon_1685986977889/work/src/geometryTransform.cu(315): error: identifier "cudaBindTextureToArray" is undefined

```

see  https://github.com/conda-forge/cudadecon-feedstock/pull/18

this issue is to track support for cuda12 before trying the feedstock again later

**Comment by jakirkham** [comment]
Should add we have seen this on both Linux & Windows

**Comment by tlambert03** [comment]
closed by #29
